src/tools/CMakeLists.txt

```diff
@@ -148,3 +148,7 @@ endif()
 add_subdirectory(immutable_object_cache)
 add_subdirectory(ceph-dencoder)
 add_subdirectory(erasure-code)
+
+if(WITH_SPDK)
+  add_subdirectory(nvme-rbd-gw)
+endif(WITH_SPDK)
```

Comment:
Can we call it `nvmf-rbd-gw` instead, so that it's clear that this is for NVMe Over Fabrics? Sorry for the bikeshedding.

Reply:
yep, sure thing

Reply:
... or `rbd-nvmf-gw` since it would align w/ our iSCSI gateway naming scheme and it is a NVMeOF (target) gateway